src/Data/TypeRepMap/Internal.hs

```diff
@@ -162,6 +162,20 @@ delete :: forall a (f :: KindOf a -> Type) . Typeable a => TypeRepMap f -> TypeR
 delete = fromTriples . deleteByFst (typeFp @a) . toTriples
 {-# INLINE delete #-}
 
+{- |
+Update a value at a specific key with the result of the provided function. When
+the key is not a member of the map, the original map is returned.
+
+>>> trmap = fromList @(TypeRepMap Identity) [WrapTypeable $ Identity "a"]
+>>> lookup @String $ adjustWithKey (fmap (++ "ww")) trmap
+Just (Identity "aww")
+-}
+adjustWithKey :: forall a f . Typeable a => (f a -> f a) -> TypeRepMap f -> TypeRepMap f
```

Comment:
This version of the function should be called `adjust` (not `adjustWithKey`) because the mapper function is `(f a -> f a)`, not `(Typeable a => f a -> f a)`.

And now that I think about it, we don't need a separate `adjustWithKey`.

So it's sufficient to simply rename this to `adjust`.